### ISSUE TITLE
Including data with package

### ISSUE THREAD

**Issue by machow** [bradlindblad/schrutepy#42]
Hello--thanks for this package (and the handy R version too!). I noticed that the R package contains the data itself, while the python package (as installed) fetches it from github when you run `load_schrute`. Any interest in making it like the R package, so the data is saved on install, and then loaded from disk in `load_schrute`?

Happy to open a PR. The changes would essentially be...

* moving the data into the `schrutepy` folder
* adding a MANIFEST.in file with the name of the data file
* adding include_package_data=True in setup.py

Here's tiny [python data package example](https://github.com/machow/intro-to-siuba/tree/master/utils/music_top200)

**Comment by bradlindblad** [comment]
Yes, big interest. I'm not very well-versed in creating python packages so a PR would be much appreciated. Excited to see how you do it!

**Comment by bradlindblad** [comment]
I was going to do this if you weren't going to PR this @machow

**Comment by machow** [comment]
Hey! I'm off this week, and don't mind trying a PR right now :). Do you know what the minimum version of python you're looking to support is? 

For context, the most recent pandas is python 3.8+, and numpy has a handy table for its recommendations of minimum python support: https://numpy.org/neps/nep-0029-deprecation_policy.html

(Somewhere around Python 3.7, the machinery I mentioned in my original comment was moved into python's builtin importlib library, but the original approach should still work with setuptools installed, which I think this package does)
